### DISCUSSION TITLE
Module has no exports on web

### DISCUSSION THREAD

**Discussion by Raduc4** [software-mansion/react-native-svg#2401] [Q&A]
Hi, I have upgraded the library an error ocured, don't know how to solve it.
I'm using expo and upgraded react native svg from `13.9.0` to `15.5.0`

```
WARNING in ./node_modules/react-native-svg/lib/module/web/utils/prepare.js:57:8
export 'hasTouchableProperty' (imported as 'hasTouchableProperty') was not found in '../../../../../..' (module has no exports)
  55 |     ref?: unknown;
  56 |   } = {
> 57 |     ...(hasTouchableProperty(props)
     |        ^
  58 |       ? {
  59 |           onStartShouldSetResponder:
  60 |             self.touchableHandleStartShouldSetResponder,

WARNING in ./node_modules/react-native-svg/lib/module/web/utils/prepare.js:79:26
export 'parseTransformProp' (imported as 'parseTransformProp') was not found in '../../../../../..' (module has no exports)
  77 |
  78 |   // we do it like this because setting transform as undefined causes error in web
> 79 |   const parsedTransform = parseTransformProp(transform, props);
     |                          ^
  80 |   if (parsedTransform) {
  81 |     clean.transform = parsedTransform;
  82 |   }
WARNING in ./node_modules/react-native-svg/lib/module/web/utils/prepare.js:83:34
export 'parseTransformProp' (imported as 'parseTransformProp') was not found in '../../../../../..' (module has no exports)
  81 |     clean.transform = parsedTransform;
  82 |   }
> 83 |   const parsedGradientTransform = parseTransformProp(gradientTransform);
     |                                  ^
  84 |   if (parsedGradientTransform) {
  85 |     clean.gradientTransform = parsedGradientTransform;
  86 |   }

WARNING in ./node_modules/react-native-svg/lib/module/web/utils/prepare.js:87:33
export 'parseTransformProp' (imported as 'parseTransformProp') was not found in '../../../../../..' (module has no exports)
  85 |     clean.gradientTransform = parsedGradientTransform;
  86 |   }
> 87 |   const parsedPatternTransform = parseTransformProp(patternTransform);
     |                                 ^
  88 |   if (parsedPatternTransform) {
  89 |     clean.patternTransform = parsedPatternTransform;
  90 |   }
```

**Comment by bohdanprog** [comment]
Hello @Raduc4,
Can you give a simple example of how to reproduce that problem? (may you create a repository with that example)Thank you

**Reply by migueldaipre** [reply]
> Hello @Raduc4, Can you give a simple example of how to reproduce that problem? (may you create a repository with that example)Thank you

Can you take a look at my [PR](https://github.com/software-mansion/react-native-svg/pull/2430) please?

**Reply by bohdanprog** [reply]
@migueldaipre  Can you tell me how we can reproduce that issue, please?

**Reply by migueldaipre** [reply]
> @migueldaipre Can you tell me how we can reproduce that issue, please?

I have a simple project with react-native and react-native-web without expo.

And a `webpack.config.js` configuration file that handles the bundle for the web. 

I'll try to create a minimal reproducer and send here

**Comment by migueldaipre** [comment]
Same issue here, i'll try to create a reproducer

**Reply by bohdanprog** [reply]
Hello @migueldaipre,
Here is a screen recording from my screen without any changes.
Can you tell me how we can reproduce that issue, please?
https://github.com/user-attachments/assets/b4bfa3d4-1013-45f7-af51-7ae8eedf1c11

**Comment by a-eid** [comment]
same issue unfortunately for me as well.


<img width="1512" alt="Screenshot 2024-10-18 at 2 48 18 AM" src="https://github.com/user-attachments/assets/c455e4dd-d125-4159-a078-eeefc4e31ea0">
